src-electron/db/query-package.js

```diff
@@ -71,6 +71,25 @@ async function getPackageByParent(db, parentId) {
     .then((rows) => rows.map(dbMapping.map.package))
 }
 
+/**
```

Comment:
@Amirreza-Ghasemkhani , 
I don't think this is part of our vue tour code. 
Did this get accidentally introduced as part of a rebase/merge?

Reply:
I tried to fix vuetour on matter and for that I need allpackages or uioption I tried to fix that but we need work on some of apies so after discussion we decide to work on this problem in another branch